api/src/main/java/marquez/db/JobDao.java

```diff
@@ -107,10 +115,10 @@
           + "LIMIT :limit OFFSET :offset")
   List<Job> findAll(String namespaceName, int limit, int offset);
 
-  @SqlQuery("SELECT count(*) FROM jobs AS j")
+  @SqlQuery("SELECT count(*) FROM jobs_view AS j")
   int count(String namespaceName);
```

Comment:
_Minor:_ Not related to this PR at all, but `namespaceName` isn't used in the SQL